moveit_core/CMakeLists.txt

```diff
@@ -44,8 +44,16 @@ endif()
 
 find_package(fcl QUIET)
 if (fcl_FOUND)
-  set(LIBFCL_LIBRARIES ${FCL_LIBRARIES})
-  set(LIBFCL_INCLUDE_DIRS ${FCL_INCLUDE_DIRS})
+  # convert cmake target to catkin variables for catkin_package DEPENDS
```

Comment:
There not really *catkin* variables, are they?

Reply:
true.